tests/main/assumes/task.yaml

```diff
@@ -0,0 +1,44 @@
+summary: Test that assumes work correctly
+
+execute: |
+    echo "Try to install an impossible version"
+    if snap install --edge test-snapd-assumes 2>stderr.txt; then
```

Comment:
Nitpick, nothing removes `stderr.txt`

Reply:
The restore code automatically cleans the current working directory.